### PR TITLE
update yarn sources

### PR DESCRIPTION
**Pull request by jywarren**
Related to #1215 #3571 potentially https://github.com/publiclab/plots2/pull/4099/files

We may be able to just remove a number of these based on this analysis:

https://github.com/publiclab/plots2/blame/3f0bdf2bb12a89d72a05a3d52738f2d8d41d3aeb/bower.json

### CODE REVIEW

**Comment by jywarren** [comment]
Major reductions!  Can prob remove flot soon too -- it's only used on https://publiclab.org/contributors now.

**Comment by jywarren** [comment]
Hi @gauravano i'm working to get a bunch of yarn deps upgraded and one was `i18n-js` - do you think you can try `3.x.x` to see if it works as you've been using it? There are actually no available versions at `2.x.x` unless we point directly at the GitHub repository and I'd prefer just to come up to date. Thanks!

https://github.com/fnando/i18n-js

**Comment by gauravano** [comment]
Hey @jywarren, thanks for mentioning. I definitely want to test newer version. Can we do it after testing and merging #4751 ? Thanks!

File: package.json [hunk 1]
```diff
@@ -21,51 +21,43 @@
...
+    "bootstrap-datepicker": "^1.6.0",
+    "bootstrap-tokenfield": "^0.12.0",
+    "chart.js": "^2.7.0",
+    "cytoscape": "^3.3.2",
+    "file-saver": "^2.0.0",
+    "flot": "^0.8.3",
+    "font-awesome": "^4.5.0",
+    "horsey": "^3.0.0",
+    "i18n-js": "^3.2.1",
+    "ics": "nwcell/ics.js^0.2.0",
```
Comment (jywarren) [inline package.json]: Tried changing this to `ics` via npm but that's actually a different project https://github.com/adamgibbons/ics (maybe better maintained)

whereas we're using: https://github.com/nwcell/ics.js

**Comment by jywarren** [comment]
This seems to be running smoothly at unstable. @gauravano if you think the i18js works, you can go ahead and merge this!

**Pull request merged**